src/v/cluster/self_test/cloudcheck.cc

```diff
@@ -228,7 +228,7 @@ ss::future<self_test_result> cloudcheck::verify_upload(
   cloud_storage_clients::object_key key,
   const std::optional<iobuf>& payload) {
     auto result = self_test_result{
-      .name = _opts.name, .info = "Put", .test_type = "cloud_storage"};
+      .name = _opts.name, .info = "Put", .test_type = "cloud"};
```

Comment:
I'm curious how this manifested itself. Did specifying just the cloud test previously not work?

Reply:
No, the cloud test worked fine and there was no behavioural bug here. The Go side `adminapi.CloudcheckTagIdentifier` was never compared against the `test_type` before, but I am rectifying its value to perform the check on `L167` in `status.go` now.

Reply:
Got it, thanks

In the future it may be nice to include those details in the commit message, just call out that a subsequent commit will use this behavior, even if this commit alone doesn't have major functional changes